code/modules/mob/living/living.dm

```diff
@@ -437,13 +437,14 @@ Sorry Giacom. Please don't be mad :(
 
 
 /mob/living/proc/electrocute_act(shock_damage, obj/source, siemens_coeff = 1, safety = 0, tesla_shock = 0)
-	adjustFireLoss(shock_damage)
-	visible_message(
-		"<span class='danger'>[src] was shocked by \the [source]!</span>", \
-		"<span class='userdanger'>You feel a powerful shock coursing through your body!</span>", \
-		"<span class='italics'>You hear a heavy electrical crack.</span>" \
-	)
-	return shock_damage
+	if(shock_damage > 0)
```

Comment:
The reason why I included this in the first place is SPECIFICALLY so that borgs don't get the "You be electrocuted" message when messing with an empty power grid.  changing the > to >= would defeat that.